tests/testthat/test-sce_to_anndata.R

```diff
@@ -20,7 +20,16 @@ metadata(sce) <- list(
   sample_metadata = data.frame(
     library_id = "library1",
     sample_id = "sample1"
-  )
+  ),
+  test_dataframe = tibble::tibble(
+    "char_col" = c("name1", NA),
+    "num_col" = 1:2,
+    "na_col" = c(NA, NA),
+    "list_col" = list(a = 1, b = "2")
```

Comment:
Nothing wrong with this, but it doesn't really fit how I think of list columns in practice...
```suggestion
    "list_col" = list(list(a = 1, b = "foo"), list(a = 2, b = "bar"))
```